training/configs/training_config.yaml

```diff
@@ -1,7 +1,8 @@
 ### General
 # in debug_mode no agent directories will be created and no models will be saved
-# further no wandb logging and fake (simulated) multiprocessing for simpler debugging 
-debug_mode: false
+# further no wandb logging and fake (simulated) multiprocessing for traceback
+debug_mode: true
+robot: "jackal"
```

Comment:
wird der nicht in start_training.launch angegeben?

Reply:
Sollte eigentlich nicht

Reply:
Verändert nichts auf der Simulationsseite

Reply:
Was macht dann das Argument model in der start_training.launch